Doc/library/importlib.metadata.rst

```diff
@@ -73,7 +73,7 @@ something into it:
 
 .. code-block:: shell-session
 
-    $ python3 -m venv example
```

Comment:
I think we should leave `python3` here as it is an instruction how to create a virtual environment. I didn't change it on purpose.

Reply:
I agree since older systems may still have Python 2.7 set a `python`.